crates/hotshot/src/traits/networking/libp2p_network.rs

```diff
@@ -268,53 +267,19 @@ impl<TYPES: NodeType> TestableNetworkingImplementation<TYPES>
                 )
                 .expect("Failed to sign DHT lookup record");
 
-                // we want the majority of peers to have this lying around.
+                // We want 2/3 of the nodes to have any given record in the DHT
                 let replication_factor = NonZeroUsize::new(2 * expected_node_count / 3).unwrap();
-                let config = if node_id < num_bootstrap as u64 {
-                    NetworkNodeConfigBuilder::default()
-                        // NOTICE the implicit assumption that bootstrap is less
-                        // than half the network. This seems reasonable.
-                        .mesh_params(Some(MeshParams {
-                            mesh_n_high: expected_node_count,
-                            mesh_n_low: 5,
-                            mesh_outbound_min: 3,
-                            // the worst case of 7/2+3 > 5
```

Comment:
This spot removes bootstrap node logic when starting tests. We don't need it because we have been treating every node as potentially "bootstrap"